input/fsh/modules/assay.fsh

```diff
@@ -0,0 +1,12 @@
+Profile: NcpiAssay
+Parent: ServiceRequest
+Id: ncpi-informatics-assay-request
+Title: "NCPI Informatics Assay Request"
+Description: "Profile to represent informatics assay requests within NCPI, such as WGS, RNA-seq, etc."
+
+* status 1..1
+* intent = #order
+* code 1..1 MS
+* code from $loinc (preferred)
```

Comment:
If you want to specify a Code System, this should be * code from system $loinc (preferred)   According to the qa report, what you have there is expected to be a Value Set.